testgrid/tgrun/hack/test-spec.yaml

```diff
@@ -1,6 +1,7 @@
 - name: hack
   numPrimaryNodes: 1
   numSecondaryNodes: 0
+  flags: "ha"
```

Comment:
this flag isn't being used by the bash script

Reply:
it is being used here

https://github.com/replicatedhq/kURL/blob/c54f2d37ce4f537e6f7212eb17f234392baa5c0d/testgrid/tgrun/pkg/runner/vmi/vmi.go#L122

https://github.com/replicatedhq/kURL/blob/98ac102b74ab9c043dc2533ca4d87afff820922e/testgrid/tgrun/pkg/runner/vmi/embed/runcmd.sh#L62

Reply:
I've fixed that and tested it with the following flags 
```
flags: "labels=disktype=ssd,gpu=enabled exclude-builtin-host-preflights"
```